src/codegen/compile.ml

```diff
@@ -4312,6 +4301,19 @@ module RTS_Exports = struct
     E.add_export env (nr {
       name = Wasm.Utf8.decode "rts_trap";
       edesc = nr (FuncExport (nr rts_trap_fi))
+    });
+
+    let stable64_write_moc_fi =
+      if E.mode env = Flags.WASIMode then
+        E.add_fun env "stable64_write_moc" (
+            Func.of_body env ["to", I64Type; "from", I64Type; "len", I64Type] [] (fun env ->
+                G.nop
```

Comment:
Maybe trap here? This shouldn’t ever be called, right?

Reply:
Totally!

Reply:
Done: 519b8646d